web/src/games/picnicGo/cards.ts

```diff
@@ -0,0 +1,145 @@
+import { cardTypes, ICardDefinition, IPlayer } from './types';
+
+export const cardDefinitions: ICardDefinition[] = [
+  {
+    id: 'sandwichChicken',
```

Comment:
You could use an enum instead of string here, which will give you type safety. Also, you don't need to store the name in the state, you can have a function that given the Id, outputs the name.

Reply:
Just overhauled most of the card system - it now uses enums and gives the name in a separate function just as you suggested. As a bonus, I also got rid of the deck property in the global game object, seeing as it's not used for anything.